snapshot-file-test/src/main/java/com/hazelcast/jet/tests/snapshot/file/GeneratedFilesVerifier.java

```diff
@@ -122,6 +130,21 @@ private void verifyQueue() {
         }
     }
 
+    private void archiveLoadedFile(Path path) throws IOException {
+        // add suffix to not override files which can have the same name (it is mostly for dynamic cluster)
+        String filename = path.toFile().getName() + "_" + UuidUtil.newUnsecureUuidString().substring(0, 8);
+        Path moveTo = filesinkArchiveCurrentSubdirectory.resolve(filename);
+        Files.move(path, moveTo);
+    }
+
+    private void prepareOrCheckCurrentArchiveDirectory() throws IOException {
+        long filename = counter / ARCHIVE_DIR_COUNTER_LIMIT;
+        filesinkArchiveCurrentSubdirectory = filesinkArchiveDirectory.resolve(Long.toString(filename));
```

Comment:
the variable name is too long, even `currentSubdirectory` should be enough

filesinkArchiveDirectory -> archiveDirectory
filesinkArchiveCurrentSubdirectory -> currentSubdirectory
filesinkDirectory -> testDirectory

This really helps while reading code